extensions/mssql/src/features.ts

```diff
@@ -92,6 +103,38 @@ export class AccountFeature implements StaticFeature {
 		return params;
 	}
 
+	protected async refreshToken(request: contracts.RefreshTokenParams): Promise<contracts.TokenRefreshedParams> {
+
+		// find account
+		const accountList = await azdata.accounts.getAllAccounts();
+		const account = accountList.find(a => a.key.accountId === request.accountId);
+		if (account) {
+			console.log(localizedConstants.failedToFindAccount(request.accountId));
```

Comment:
You shouldn't be logging localized text to the console - that's for debugging so should always be in english.